### DISCUSSION TITLE
Erro NFSE python 3.9

### DISCUSSION THREAD

**Discussion by sidneiJr1984** [TadaSoftware/PyNFe#324] [Q&A]
Boa tarde, para emissao de NFSE é indicado a instalação do suds-jurko porém no python 3.9 ela já está descontinuada

Existe alguma correção para ele?

**Comment by juniortada** [comment]
@sidneiJr1984 a emissão de NFS-e está bem desatualizada.
Será preciso refatorar grande parte do código.
